apps/web/src/pages/auth/components/LoginForm.tsx

```diff
@@ -48,40 +51,59 @@ export function LoginForm({ email, invitationToken }: LoginFormProps) {
     }
   >((data) => api.post('/v1/auth/login', data));
 
-  useEffect(() => {
-    (async () => {
-      if (!tokenInQuery) {
-        return;
-      }
+  const handleLoginInUseEffect = async () => {
+    // if currentUser is true, it means user exists, then while accepting invitation, InvitationPage will handle accept this case
```

Comment:
This indicates a routing issue as the invitation handling should be done in one place, not between pages. It's OK for now but we will have to get back to this.